segmentstore/server/src/main/java/io/pravega/segmentstore/server/logs/InMemoryLog.java

```diff
@@ -17,24 +17,37 @@
 
 import com.google.common.annotations.VisibleForTesting;
 import io.pravega.common.util.BlockingDrainingQueue;
+import io.pravega.segmentstore.server.logs.health.InMemoryLogHealthContributor;
 import io.pravega.segmentstore.server.logs.operations.Operation;
 import javax.annotation.concurrent.GuardedBy;
+
+import io.pravega.shared.health.HealthConnector;
+import io.pravega.shared.health.HealthContributor;
 import lombok.AccessLevel;
 import lombok.Getter;
 
+import java.util.Queue;
+
 /**
  * {@link BlockingDrainingQueue} implementation for {@link Operation}s. Prevents adding {@link Operation}s out of order.
  */
-public class InMemoryLog extends BlockingDrainingQueue<Operation> {
+public class InMemoryLog extends BlockingDrainingQueue<Operation> implements HealthConnector {
     /**
      * The last sequence number added. This field is only accessed in {@link #addInternal}, which is guaranteed to be
      * executed while holding the base class' lock, hence no need for extra synchronization here.
      */
-    @Getter(AccessLevel.PACKAGE)
     @GuardedBy("AbstractDrainingQueue.this.lock")
     @VisibleForTesting
+    @Getter(AccessLevel.PACKAGE)
```

Comment:
This change seems unnecessary.

Reply:
This is used to export the `lastSequenceNumber` for the contributor details.